README.md

```diff
@@ -98,22 +103,27 @@ Options:
 -m:	Minimise output file size by not saving vertex and facet information
```

Comment:
I notice that `-c, --cubic-interpolation` are missing from this printout. Although #217 will make this comment redundant anyhow.